os/drivers/testcase/kernel_testcase_drv.c

```diff
@@ -120,6 +133,31 @@ static int kernel_test_drv_ioctl(FAR struct file *filep, int cmd, unsigned long
 		break;
 #endif
 
+#ifdef CONFIG_EXAMPLES_MPU_TEST
+	case TESTIOC_MPUTEST:
+		ret = OK;
+		struct mputest_arg_s *obj = (struct mputest_arg_s*)arg;
```

Comment:
arg is unsigned long.
is it right?

Reply:
Yes. Using arg like this is ok. It is common way of passing argument to ioctl.
I have added the null check.